### PR TITLE
Fix some typos in docs

### PR DESCRIPTION
**Pull request by p4v4n**

### CODE REVIEW

File: doc/modules/ROOT/pages/nrepl-api/ops.adoc [hunk 1]
```diff
@@ -309,7 +309,7 @@ Required parameters::
 {blank}
 
 Optional parameters::
-* `:class` A Java class. If ``:ns`` is passed, it will be used for fully-qualifiying the class, if necessary.
+* `:class` A Java class. If ``:ns`` is passed, it will be used for fully-qualifying the class, if necessary.
```
Comment (vemv) [inline doc/modules/ROOT/pages/nrepl-api/ops.adoc:312]: .adoc files auto-generated, please fix the source instead and re-run `lein docs`

**Review by vemv — COMMENTED** [review]
Looking good, needs a tweak though

Thanks!

File: doc/modules/ROOT/pages/nrepl-api/ops.adoc (hunk 1 above)
Comment (p4v4n) [reply doc/modules/ROOT/pages/nrepl-api/ops.adoc:312]: Thanks. Made the changes.

**Comment by vemv** [comment]
Thanks much!

**Pull request merged**